exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporter.java

```diff
@@ -158,6 +157,10 @@ static Span generateSpan(SpanData spanData, Endpoint localEndpoint) {
     return spanBuilder.build();
   }
 
+  private static String getValueOrReturnEmpty(String value) {
```

Comment:
```suggestion
  private static String nullToEmpty(String value) {
```

Just small nit, we're not really "getting the value" from anything here